### PR TITLE
Sequential OED with sigmoid model

### PR DESCRIPTION
**Pull request by ae-foster**

### CODE REVIEW

File: examples/contrib/oed/ba/guide.py [hunk 1]
```diff
@@ -150,7 +157,8 @@ def tensorized_matrix_inverse(M):
         inv = inv/det.unsqueeze(-1).unsqueeze(-1)
         return inv
     else:
-        raise NotImplemented()
+        b = [t.inverse() for t in torch.functional.unbind(M)]
+        return torch.stack(b)
```
Comment (ae-foster) [inline examples/contrib/oed/ba/guide.py]: Clean this up- move to `util.py`

File: examples/contrib/oed/sequential_oed_sigmoid_lm.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    response_mean = rmv(design, w)
+
+    base_dist = dist.Normal(response_mean, torch.tensor(1.)).independent(1)
+    k = k.expand(response_mean.shape)
+    transforms = [AffineTransform(loc=0., scale=k), SigmoidTransform()]
+    response_dist = dist.TransformedDistribution(base_dist, transforms)
+    return pyro.sample("y", response_dist)
+
+
+def svi_guide(design):
```
Comment (ae-foster) [inline examples/contrib/oed/sequential_oed_sigmoid_lm.py]: Unused

File: examples/contrib/oed/sequential_oed_sigmoid_lm.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+    w1_mean = pyro.param("w1_mean", torch.zeros(batch_shape + (2,)))
+    w1_sds = softplus(pyro.param("w1_sds", -5.*torch.ones(batch_shape + (2,))))
+    pyro.sample("w1", dist.Normal(w1_mean, w1_sds).independent(1))
+    w2_mean = torch.tensor(0.)
+    w2_sds = torch.tensor([1.]*5 + [10.]*5)
+    pyro.sample("w2", dist.Normal(w2_mean, w2_sds).independent(1))
+
+
+def learn_posterior(y, d, model, svi_guide):
```
Comment (ae-foster) [inline examples/contrib/oed/sequential_oed_sigmoid_lm.py]: Unused

File: examples/contrib/oed/ba/guide.py (hunk 1 above)
Comment (ae-foster) [reply examples/contrib/oed/ba/guide.py]: Leave until after merge- clean up in a separate step
